packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/ccn/1486/indemnite.js

```diff
@@ -0,0 +1,178 @@
+import { round, sum } from "../../../common/math";
+
+const COEFF_ETAM_0 = 1 / 4;
+const COEFF_ETAM_1 = 0.3;
+const MAX_MONTH_INDEMNITE_ETAM = 10;
+const COEFF_IC = 1 / 3;
+const MAX_MONTH_INDEMNITE_IC = 12;
+const COEFF_CEI = 1 / 5;
+const MAX_MONTH_INDEMNITE_CEI = 7;
+
+export const ERROR_LABEL =
+  "Aucune indemnité de licenciement conventionnelle n’est prévue en deçà de 2 ans d’ancienneté.";
+
+function getAncienneteConventionnelle({
+  brancheCategorie,
+  ancienneteLegale,
+  hasBrancheContrat,
+  brancheContrat,
+  brancheAncienneteCE,
+  brancheAncienneteEnqueteur
+}) {
+  let ancienneteConventionnelle = 0;
+  if (brancheCategorie === "CEI" || brancheCategorie === "CENI") {
+    ancienneteConventionnelle +=
+      parseFloat(brancheAncienneteCE) / 12 +
+      parseInt(brancheAncienneteEnqueteur); // We only accept round years
```

Comment:
if you only to floor a value, use Math.floor() so the intent is clearer also round value should be handled in the step

Reply:
👌